File: 3701.compute-alternating-sum.java
/*
 * @lc app=leetcode id=3701 lang=java
 *
 * [3701] Compute Alternating Sum
 */

// @lc code=start
class Solution {
    public int alternatingSum(int[] nums) {
        int result = 0;
        for (int i = 0; i < nums.length; i++) {
            if (i % 2 == 1) {
                result -= nums[i];
            }
            else {
                result += nums[i];
            }
        }
        return result;
    }
}
// @lc code=end

File: 3701.compute-alternating-sum.py
#
# @lc app=leetcode id=3701 lang=python
#
# [3701] Compute Alternating Sum
#

# @lc code=start
class Solution(object):
    def alternatingSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        result = 0
        for i in range(len(nums)):
            if (i % 2):
                result -= nums[i]
            else:
                result += nums[i]
        return result
        
# @lc code=end

